lib/presentation/widget/myspace/my_space_widget.dart

```diff
@@ -208,14 +264,27 @@ class _MySpaceWidgetState extends State<MySpaceWidget> {
           ],
         ),
       ) : null,
-      trailing: IconButton(
-        icon: SvgPicture.asset(
-          imagePath.icContextMenu,
-          width: 24,
-          height: 24,
-          fit: BoxFit.fill,
-      ), onPressed: () => mySpaceViewModel
-          .openContextMenu(context, document, contextMenuActionTiles(context, document))),
+      trailing: StoreConnector<AppState, MySpaceState>(
```

Comment:
I prefer use `StoreConnector` to rebuild `ListView` instead of element of each item. Because, when we works with 10000.. items or more. It's create a lot of connector to Store. And it's seem terrible '_'

Just my guest base on my knowledge. Plz help me test it. About use `StoreConnector` to rebuild `ListView`, indeed we just rebuild items are displaying on screen, I think the performance is good ^^

Reply:
I think it's better to put StoreConnector to lowest elements and make the viewModel really convenient since it will be rebuilt everytime this viewModel changed.
In my case, it would be better to have 
```
      trailing: StoreConnector<AppState, SelectMode>(
        converter: (store) => store.state.mySpaceState.selectMode,
        builder: (context, selectMode) {
```
However you're right, I put a storeConnector just for the select mode I could have just give this in parameter. 

About performance, I don't think it will be a problem since we don't have a lot of documents lists at the same time